internal/tools/pipeline/dbclient/op_pipeline.go

```diff
@@ -405,16 +406,16 @@ func (client *Client) PageListPipelines(req *pipelinepb.PipelinePagingRequest, o
 	}
 	baseSQL.Where(tableFieldName((&spec.PipelineBase{}).TableName(), "is_snippet")+" = ?", req.IncludeSnippet)
 	if req.StartTimeBegin != nil && !req.StartTimeBegin.AsTime().IsZero() {
-		baseSQL.Where(tableFieldName((&spec.PipelineBase{}).TableName(), "time_begin")+" >= ? ", req.StartTimeBegin)
+		baseSQL.Where(tableFieldName((&spec.PipelineBase{}).TableName(), "time_begin")+" >= ? ", req.StartTimeBegin.AsTime().In(time.Local))
```

Comment:
I think you should wrap a method for pb time, see `pkg/common/pbutil` pkg.

Reply:
done